AllfleXML/FlexOrder/FlexOrder.cs

```diff
@@ -54,7 +57,15 @@ public static Document Import(XDocument document)
 
         public static XDocument Export(this OrderHeader order)
         {
-            return Export(new Document { OrderHeaders = new List<OrderHeader> { order } });
+            var result = new XDocument();
```

Comment:
Dont we want it to do both?  Export as just an Order and also as a Document?

Reply:
Because the api.allflexusa.com did not follow this specification, this is a nessessity

Reply:
you can't post a document to the api, you can only post an orderHeader

Reply:
ah, wait, I see.   We do still have the ability to Export a document if need be.